packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -102,6 +106,20 @@ export class BlockList extends Component {
 		);
 	}
 
+	getExtraData() {
+		if (
```

Comment:
I think it's worth to destruct props:

`const { renderFooterAppender, parentWidth } = this.props;`